integration/unfork/gcloud-sqlproxy/metadata.yaml

```diff
@@ -1,3 +1,3 @@
 args: ["--prefer-git"]
 upstream: https://github.com/rimusz/charts/tree/44e2acc78cd7b621ca306661770828f32e9a2820/stable/gcloud-sqlproxy
-fork: https://github.com/replicatedhq/test-charts/tree/859cd8dbed96e952a2c6901392e6fe341e56bdf3/gcloud-sqlproxy-forked
+fork: https://github.com/replicatedhq/test-charts/tree/a0d1eefbf133fa2776164d0be2ba2a382811c16c/gcloud-sqlproxy-forked
```

Comment:
Why's this changing? Not that I'm objecting, since the rest of the test seems to have stayed the same, but not something I expected to see

Reply:
i didn't merge it and then squash and merged haha